src/AppBundle/Resources/views/Odr/Index/_report_row.html.twig

```diff
@@ -20,10 +20,7 @@
                 {% endif %}
             </p>
             <p class="text-secondary soft-half--top">
-                {% if reportActive %}
```

Comment:
@sheanie reportActive (as far as I remember) was the current report for clients (among the potential already-submitted ones). Is it not needed anymore with this feature ?